### PR TITLE
PR_Templates.md & feature_request_template.md

### PR DESCRIPTION
**Pull request by ritiksoni00**
Fixes: #6 [Add Issue and PR template]

### CODE REVIEW

File: .github/Pull_request_Template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Related Issuse or bug
+  - Info about Issue or bug
+
+# Fixes
+  - [issue number that will be closed through this PR]
```
Comment (Kajol-Kumari) [inline .github/Pull_request_Template.md:5]: Hey @ritiksoni00 please provide this full line here `Fixes: #[issue number that will be closed through this PR]`

**Comment by ritiksoni00** [comment]
> Minor change, Rest looks good to me :)
> Also from next time please make sure to create a branch before starting to work on an issue.

Ohkayy next time ill keep in mind that

**Comment by Kajol-Kumari** [comment]
LGTM :)

This can be merged now @harsh-9in

**Comment by ritiksoni00** [comment]
Thanyuu! @Kajol-Kumari  & @harsh-9in ! looking forward...

**Pull request merged**